setup.cfg

```diff
@@ -0,0 +1,44 @@
+[metadata]
+name = pytest-astropy
+url = https://astropy.org
```

Comment:
Not sure what's used for other astropy packages, but I find useful to have a more precise URL here, like the github url. Otherwise it is sometimes annoying to find the repository.

Reply:
We can [add named links with the `project_urls` option](https://packaging.python.org/guides/distributing-packages-using-setuptools/#project-urls) - e.g. [this](https://github.com/HypothesisWorks/hypothesis/blob/11c7c08d1a1b5208890f4bb7e1bf9359fcff80d1/hypothesis-python/setup.py#L84-L88) renders [like so](https://pypi.org/project/hypothesis/).

Reply:
That's very cool! For now I'll just link to the repo since for this specific package there aren't e.g. separate docs and so on, but I'll definitely be using this for other packages.

Reply:
Oooo that is very convenient indeed, @zac-hd , thanks! @astrofrog , any plans to use `project_urls` in your APE 17 PR over at `astropy`?

Reply:
@pllim - yes I can try and do that